frontends/precinct-scanner/src/screens/election_manager_screen.tsx

```diff
@@ -156,10 +158,16 @@ export function ElectionManagerScreen({
             onChange={changeAppPrecinctSelection}
             large
           >
-            <option value="" disabled>
-              Select a precinct for this device…
-            </option>
-            <option value={ALL_PRECINCTS_OPTION_VALUE}>All Precincts</option>
+            {!precinctSelectionValue && (
```

Comment:
What does this change do?

Reply:
Hides the `Select a precinct for this device...` text when the precinct is already set, because it's just a disabled option in the list. A bit unrelated, and happy to reverse.

Reply:
Cool, that seems fine